srcpkgs/libibmad/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'libibmad'
+pkgname=libibmad
+version=1.3.12
+revision=1
+wrksrc="OFED-3.18-2/SRPMS"
+build_wrksrc="libibmad-1.3.12"
+build_style=gnu-configure
+hostmakedepends="rpm cpio"
+makedepends="rdma-core-devel"
+depends="rdma-core"
+short_desc="OpenFabrics Alliance InfiniBand diagnostic and management library"
+maintainer="Rich Gannon <rich@richgannon.net>"
+license="GPL-2.0-or-later"
+homepage="https://openfabrics.org"
+distfiles="${homepage}/downloads/OFED/ofed-3.18-2/OFED-3.18-2.tgz"
```

Comment:
I think it’s better to not use `${homepage}` here, just write it out. It’s much easier to read and copy+paste the link.

Reply:
Corrected.